index.html

```diff
@@ -234,20 +234,20 @@ <h3>Latest news</h3>
           <!-- News box Start -->
           <div class="news-box">
             <div class="news-box-title">
-              <span>18/04/2025</span>
+              <span>26/05/2025</span>
             </div>
             <div class="news-box-content">
-	      <p>We just released a new <strong>security patch on top of Uyuni 2024.12</strong> that provides important security fixes. Check the <a href="https://lists.opensuse.org/archives/list/announce@lists.uyuni-project.org/thread/RDESYZTK7HNQD5ZF7HPCOFAWMMKO52YO/" target="_blank">announcement</a> to get all the details about the fix and instructions to install the patch. <strong>We strongly recommend you update as soon as possible.</strong></p>
+	      <p>Uyuni 2025.05 is now available with support for SL Micro 6.1 & SLE 15 SP7, new separate DB container, container images in RPMs, Health Check tool and improved CVE Audit with OVAL data! Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```

Comment:
```suggestion
	      <p>Uyuni 2025.05 is now available with support for SUSE Linux Micro 6.1 & SUSE Linux Enterprise 15 SP7, new separate DB container, container images in RPMs, Health Check tool and improved CVE Audit with OVAL data! Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```
And same for the news page.